e2e/integration/frontity-01/image.spec.js

```diff
@@ -70,9 +70,8 @@ describe("Image lazy-loading (with native lazy-load)", () => {
 
   it("should render an image without loading if it doesn't have height", () => {
     cy.scrollTo("topLeft");
-    cy.get("img:not([height])")
-      .should("not.have.attr", "loading")
-      .should("not.be.visible");
```

Comment:
Before Cypress 6.0 you could assert that `undefined` is not visible. That's not allowed anymore.